### DISCUSSION TITLE
update of formula for "joplin-cli" requested. 2.11.1 -> 2.12.1 (use same DB version as Desktop)

### DISCUSSION THREAD

**Discussion by netweaver1970** [Homebrew/discussions#4734] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.6
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3cd72905ce3d36a84994cc6c6b9ebbdafff4e922
Last commit: 5 days ago
Core tap JSON: 23 Aug 13:42 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.XxLKOuWoxf/org.xquartz:0
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.5-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: 14.3.1
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

please update the formula for the newly released version of joplin-cli (desktop and joplin-cli versions were not compatible in DB usage) .
Today that incompatibility has been rectified, by releasing a new version of joplin-cli (2.12.1)

this is the link for 2.12.1. The documentation (incl. change log) still needs updating on the website but probably will happen shortly.
https://github.com/laurent22/joplin/tree/release-2.12/packages/app-cli.

for reference, this was the original github issue : https://discourse.joplinapp.org/t/db-version-mismatch-with-latest-available-joplin-downloads/32167/4

Thanks

**Comment by ZhongRuoyu** [comment, marked as answer]
This is being done at https://github.com/Homebrew/homebrew-core/pull/140278.

Update: `joplin-cli` has been updated: https://formulae.brew.sh/formula/joplin-cli

**Reply by netweaver1970** [reply]
thanks for the swift packaging. all seems to work fine so far. I'll close the discussion.
btw, If I'm back visiting S'pore in Nov, maybe we can go for a drink, I would like to get to know the tech scene a bit more there.
PM me if interested.
